.eslintrc

```diff
@@ -58,7 +58,6 @@ rules:
   no-empty-character-class: 2
   no-self-compare: 2
   valid-typeof: 2
-  no-unused-vars: [2, { "args": "none" }]
```

Comment:
Not compatible with Typescript, however we also use the compatible version so I'm just removing the support here.